sapi/phpdbg/phpdbg_opcode.c

```diff
@@ -114,6 +114,44 @@ char *phpdbg_decode_opline(zend_op_array *ops, zend_op *op) /*{{{ */
 		break;
 	}
 
+#if 1
```

Comment:
this is up to you to decide.

Thanks. Dmitry.

On Tue, Jul 7, 2015 at 6:40 AM, Bob Weinand <notifications@github.com>
wrote:

> In sapi/phpdbg/phpdbg_opcode.c
> <https://github.com/php/php-src/pull/1393#discussion_r34005819>:
>
> > @@ -114,6 +114,44 @@ char *phpdbg_decode_opline(zend_op_array *ops, zend_op *op) /*{{{ */
> >  		break;
> >  	}
> >
> > +#if 1
>
> Do we want to make it #if 0 when merging?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/php/php-src/pull/1393/files#r34005819>.
>